vendor/github.com/aws/aws-sdk-go/service/elasticache/api.go

```diff
@@ -2572,6 +2572,9 @@ type CacheCluster struct {
 	// node.
 	ConfigurationEndpoint *Endpoint `type:"structure"`
 
+	// The DNS name of the cache cluster without the port appended.
+	ClusterAddress *string `type:"string"`
```

Comment:
I did, I'm not sure if that is the correct procedure for changes within the vendor directory.